documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +138,46 @@ After receiving the "Append Setup" reply, it performs the following:
 While this is happening, the server will be periodically sending it `DataAppended` replies acking messages. Note that there can be multiple "Appends Setup" for a given TCP connection. This allows a client to share a connection when producing to multiple Segments.
 
 A client can optimize its appending by specifying a large value in it's `AppendBlock` message, as the events inside of the block do not need to be processed individually.
+
+# Segment Attribute
+
+## GetSegmentAttribute - Request
+
+| **Field**    |**Datatype**  | **Description**     |
+|-------------|----------|------|
+| `RequestId`| Long| The client-generated _ID_ that identifies a client request.|
+|  `SegmentName`| String| |
+|  `attributeId`| UUID| |
+| `delegationToken`| String| This was added to perform _auth_. It is an opaque-to-the-client token provided by the Controller that says it's allowed to make this call.|
+
+## SegmentAtrribute - Reply
+
+| **Field**    |**Datatype**  | **Description**     |
+|-------------|----------|------|
+| `RequestId`| Long| The client-generated _ID_ that identifies a client request.|
+| `Value`|Long||
+
+More information on other Request message like `updateSegmentAttribute` and Reply message like `SegmentAttributeUpdate` can be found [here](https://github.com/pravega/pravega/blob/master/shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java).
+
+# TableSegment
+
+## ReadTable - Request
+
+| **Field**    |**Datatype**  | **Description**     |
+|-------------|----------|------|
+| `RequestId`| Long| The client-generated _ID_ that identifies a client request.|
+|  `Segment`| String| The Stream Segment that was read. |
+|`delegationToken`|String| This was added to perform _auth_. It is an opaque-to-the-client token provided by the Controller that says it's allowed to make this call.|
+|`keys`|List<TableKey>|The version of the key is always set to `io.pravega.segmentstore.contracts.tables.TableKey.NO_VERSION`.|
+
+More information on `TableSegments` and its various Request messages like `MergeTableSegments`, `SealTableSegment`, `DeleteTableSegment`, `UpdateTableEntries`, `RemoveTableKeys`, `ReadTableKeys` and `ReadTableEntries` can be found [here](https://github.com/pravega/pravega/blob/master/shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java).
```

Comment:
We could change it to something on the lines
" More information on `TableSegment` Request messages... " similar to the pattern followed for Segment Attribute related Wirecommands.
(The same pattern  could be used for the TableSegment related Reply messages...)

Reply:
Edited